controller/test-files/event/list/ok-description.res.payload.golden.json

```diff
@@ -3,18 +3,19 @@
     {
       "attributes": {
         "name": "system.description",
-        "newValue": null,
+        "newValue": {
+          "content": "# Description is modified1",
+          "markup": "Markdown"
+        },
         "oldValue": null,
-        "timestamp": "0001-01-01T00:00:00Z"
+        "timestamp": "0001-01-01T00:00:00Z",
+        "workItemTypeID": "00000000-0000-0000-0000-000000000001"
```

Comment:
Note, that an event needs to know about the work item type at the given point int type. This is important so that the UI can find out how to resolve the `name` (e.g. `system.description`) into a field.

Reply:
I think I should make this a relationships really. Let me do that.

Reply:
Done in 6020b734